pin-cushion.js

```diff
@@ -330,7 +369,25 @@ class PinCushionHUD extends BasePlaceableHUD {
  * Hook on init
  */
 Hooks.on("init", () => {
+    globalThis.PinCushion = PinCushion;
```

Comment:
I don't love adding items to the global scope, but I think this is fairly safe. A future iteration might be to put this into a namespaced area such as `game.modules` or `game.pinCushion`

Reply:
This was mostly done to reduce the number of necessary edits e.g. in the AboutApp.
Moving the class someplace else instead of just `globalThis` is of course always an option.